pkgs/applications/misc/cpp-ethereum/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, jsoncpp
+, libjson_rpc_cpp
+, curl
+, boost
+, leveldb
+, cryptopp
+, libcpuid
+, opencl-headers
+, ocl-icd
+, miniupnpc
+, libmicrohttpd
+, gmp
+, mesa
+, extraCmakeFlags ? []
+}:
+stdenv.mkDerivation rec {
+  name = "cpp-ethereum-${version}";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ethereum";
+    repo = "cpp-ethereum";
+    rev = "62ab9522e58df9f28d2168ea27999a214b16ea96";
+    sha256 = "1fxgpqhmjhpv0zzs1m3yf9h8mh25dqpa7pmcfy7f9qiqpfdr4zq9";
+  };
+
+  cmakeFlags = with stdenv.lib; concatStringsSep " " (flatten [
```

Comment:
It is marked as broken in #19040 because of security issues.
The problem is mentioned here: https://lwn.net/Vulnerabilities/683992/
The current version of cryptopp on NixOS is 5.6.2, while the latest one is 5.6.4: https://www.cryptopp.com/
The issue might have been fixed.